gpu-service/jobs.py

```
@@ -0,0 +1,486 @@
+            )
+        except Exception as e:
+            logger.error(f"Failed to load ASR model: {str(e)}")
+            _asr_model = None
+    return _asr_model
+
+def safe_detect_faces(image_bgr, retry_count=3):
+    """
+    Detect faces on a BGR (OpenCV-style) image using RetinaFace.
+    The helper retries a few times to work around occasional TF/Keras padding errors.
+    """
+    import time, gc
```

Ruff (0.11.9)

Multiple imports on one line

Split imports

(E401)